pants.ini

```diff
@@ -99,15 +99,6 @@ read_from: ["%(local_artifact_cache)s"]
 write_to: ["%(local_artifact_cache)s"]
 
 
-[ivy]
-# A custom ivysettings.xml file to allow for consumption from a local .m2 repository.
-# If you don't need access to a local .m2 repository, remove this setting to use the default.
-ivy_settings: %(pants_supportdir)s/ivy/ivysettings.xml
```

Comment:
Does this mean local deploys are no longer supported? Or are they now supported by default?